knowledge-base/adr/0021-odata-url-builder.md

```diff
@@ -0,0 +1,193 @@
+# Title
+Status: in progress
+
+Decision: ??
```

Comment:
Please write down what we decided as well.

Reply:
Updated, which is not finalised.